### DISCUSSION TITLE
Consider a project name change while it's still early

### DISCUSSION THREAD

**Discussion by fireboy1919** [pyscript/pyscript#1231] [Proposals]
I notice that there's already an existing project named pyscript, and it's three years old, and already mature and widely used.

This project appears to be an alpha project that's still less than a year old.  Obviously with a big company behind it rather than one guy, and it's WAY more popular, but it is essentially taking the name from an existing project.

Are you aware of this conflict?  Have you talked about this?

**Comment by ramalho** [comment]
Can you please post a link to the project? I searched but could not find it.

**Comment by fireboy1919** [comment]
https://github.com/custom-components/pyscript
https://hacs-pyscript.readthedocs.io/en/latest/
